cli/utilities/utils.py

```diff
@@ -627,8 +627,12 @@ def check_coredump_generated(node, coredump_path, created_after):
     cmd = f"ls -Art {coredump_path} | tail -n 1"
     file_name, _ = node.exec_command(cmd=cmd, sudo=True)
 
+    # If no coredump path empty
```

Comment:
Rephrase comment as

" If the path is empty / not created, return True as no coredump has been generated"

Reply:
Thanks @pranavprakash20 
Updated